micropub.php

```diff
@@ -61,7 +61,16 @@ public static function init() {
     add_action('send_headers', array('Micropub', 'http_header'));
     add_filter('host_meta', array('Micropub', 'jrd_links'));
     add_filter('webfinger_data', array('Micropub', 'jrd_links'));
-  }
+
+    // Store MF2
+    add_filter('before_micropub', array('Micropub', 'store_mf2'), 8);
```

Comment:
Default priority is 10. Storing the MF2 in the meta should occur before the others, so I set it to 8. That means other filters will have access to it in the args array.